charts/redpanda/post_install_upgrade_job.go

```diff
@@ -55,11 +55,11 @@ func bootstrapYamlTemplater(dot *helmette.Dot) corev1.Container {
 		Resources: corev1.ResourceRequirements{
 			Limits: corev1.ResourceList{
 				corev1.ResourceCPU:    resource.MustParse("100m"),
-				corev1.ResourceMemory: resource.MustParse("25Mi"),
+				corev1.ResourceMemory: resource.MustParse("125Mi"),
```

Comment:
We should probably make these configurable in the `values.yaml`

Reply:
It was actually an active decision to make them not configurable. My logic here being "If it's correct, why would anyone need to set it to anything else?". Despite the extremely weird bug, I think I still agree with that logic. We can expand `podSpec` to allow overriding the resources of `initContainers` if you think that's safer/better? Though I'll do that at a later time rather than in this release unless you feel strongly.

Reply:
By `podSpec` I mean: https://github.com/redpanda-data/helm-charts/blob/6bf59059c8a90772d692c73baff5b700a5b939e2/charts/redpanda/helpers.go#L426

Reply:
Yeah, I'm thinking it would be good to do it a later time, definitely don't need to handle it now.

My $0.02 is that in general I'm a fan of keeping at least some hardcoded things a bit more configurable if they're going to make installation otherwise impossible, especially if we make other things around the pod running configurable (i.e. the image). I can totally imagine someone repackaging a custom image that for some reason needs slightly different resource limits.

Reply:
That's fair! I'm generally onboard as well. I've just been reticent to fully implement SMP in gotohelm and I really want to avoid stabbing any more holes into our already very inconsistent API.

Reply:
https://github.com/redpanda-data/helm-charts/issues/1571